lib/modules/manager/swift/__fixtures__/SamplePackage.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.6
```

Comment:
why do we need this change? 🤔

Reply:
Because I added an example of `exact:` syntax to this file and it is not valid in 4.2.
5.6 is the last version which supports both `exact:` and `.exact()` style.

Reply:
I think, from our side we don't care so much about right tooling version

Reply:
> Because I added an example of `exact:` syntax to this file and it is not valid in 4.2.
> 5.6 is the last version which supports both `exact:` and `.exact()` styles.

ok, makes sense. thanks for explaining. 🤗